packages/mattermost-redux/src/client/client4.ts

```diff
@@ -2309,6 +2314,20 @@ export default class Client4 {
         );
     }
 
+    setFirstAdminVisitMarketplaceStatus = async () => {
+        return this.doFetch<StatusOK>(
+            `${this.getBaseRoute()}/marketplace/first_admin_visit`,
```

Comment:
This obviously isn't something to change at this point for this feature, but this makes me wonder if we could use something like user Preferences but on a system-wide level for this type of random internal state to save us from having to add new endpoints and new data structures whenever we want something like this. We could also use it for some derived state like how we stick a few random things into the client config like the max length of posts and whether or not there are users on the server, neither of which are actual config values.